api/src/Controller/ItemController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class ItemController
+{
+    /**
+     * @Route("/api/item/list", methods={"GET","HEAD"})
```

Comment:
If we follow the REST principles, this route would be wrong. The fact that is a "list" is implicit by the formatting of the date (i.e. an array) and shall not be specified in the URL.

At the moment it is a big hard to foresee the requirements. We will need to evolve a bit the project to really understand them better and come back to the design table to define the API resources.

Anyway, I think this should be `api/items` or `api/my_first_dictionary/items` (the latter being an example of a hardcoded list to learn).

Reply:
Discussing API urls is really an exciting topic. It certainly requires being alert to do really REST methods. I finally selected the first option for this test method.

Reply:
:+1: